.gitignore

```diff
@@ -0,0 +1,2 @@
+.env
+/node_modules
```

Comment:
Please remove this line, delete the node_modules, commit and re-add this line :-) 1752 files for PR is a no-go.